ColumnReinforcement/ColumnReinforcement.py

```diff
@@ -0,0 +1,371 @@
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+__title__ = "Column Reinforcement"
+__author__ = "Suraj"
+__url__ = "https://www.freecadweb.org"
+
+from RebarDistribution import runRebarDistribution, removeRebarDistribution
+from Rebarfunc import getSelectedFace, check_selected_face
+from .SingleTie import makeSingleTieFourRebars
+from PySide import QtGui
+import FreeCAD
+import FreeCADGui
+import os
+
+
+class _ColumnTaskPanel:
+    def __init__(self, Rebar=None):
+        self.customSpacing = None
+        if not Rebar:
+            selected_obj = FreeCADGui.Selection.getSelectionEx()[0]
+            self.SelectedObj = selected_obj.Object
+            self.FaceName = selected_obj.SubElementNames[0]
+        else:
+            self.FaceName = Rebar.Base.Support[0][1][0]
+            self.SelectedObj = Rebar.Base.Support[0][0]
+        self.form = FreeCADGui.PySideUic.loadUi(os.path.splitext(__file__)[0] + ".ui")
+        self.form.setWindowTitle(
+            QtGui.QApplication.translate("RebarAddon", "Column Reinforcement", None)
+        )
+        self.form.image.setPixmap(
+            QtGui.QPixmap(
+                os.path.split(os.path.split(os.path.abspath(__file__))[0])[0]
+                + "/icons/Column_CustomConfiguration.png"
+            )
+        )
+        self.addDropdownMenuItems()
+        self.connectSignalSlots()
+        self.form.mainRebarOrientationWidget.hide()
+        self.form.x_dirRebarOrientationWidget.hide()
+        self.form.y_dirRebarOrientationWidget.hide()
+        self.Rebar = Rebar
+
+    def addDropdownMenuItems(self):
+        """ This function add dropdown items to each Gui::PrefComboBox."""
```

Comment:
No space after `"""`.
Docstring should be like this `""" This function add dropdown items to each Gui::PrefComboBox."""`